node/WsTokens.js

```diff
@@ -0,0 +1,19 @@
+module.exports = class WsTokens {
+  constructor(mysqlCon) {
+    this.con = mysqlCon;
+  }
+
+  isValid(token, userId) {
+    return new Promise((resolve, reject) => {
+      this.con.query('SELECT count(*) AS count FROM ws_tokens WHERE token = ? AND user_id = ?;',
+	  [token, userId],
+	  function (err, results) {
+         if (results[0].count <= 0) {
+           resolve(false);
+	 } else {
+           resolve(true);
+	 }
```

Comment:
This could be the following;

```suggestion
         resolve(results[0].count > 0)
```

Reply:
Done.